homeassistant/components/media_player/dlna_dmr.py

```diff
@@ -0,0 +1,542 @@
+# -*- coding: utf-8 -*-
+"""
+Support for DLNA DMR (Device Media Renderer).
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/media_player.dlna_dmr/
+"""
+
+import asyncio
+import logging
+import socket
+import urllib.parse
+from datetime import datetime
+
+import aiohttp
+import async_timeout
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.http import HomeAssistantHTTP
+from homeassistant.components.http.view import (
+    request_handler_factory, HomeAssistantView)
+from homeassistant.components.media_player import (
+    SUPPORT_PLAY, SUPPORT_PAUSE, SUPPORT_STOP,
+    SUPPORT_VOLUME_MUTE, SUPPORT_VOLUME_SET,
+    SUPPORT_PLAY_MEDIA,
+    SUPPORT_PREVIOUS_TRACK, SUPPORT_NEXT_TRACK,
+    MediaPlayerDevice,
+    PLATFORM_SCHEMA)
+from homeassistant.const import (
+    EVENT_HOMEASSISTANT_STOP,
+    CONF_URL, CONF_NAME,
+    STATE_OFF, STATE_ON, STATE_IDLE, STATE_PLAYING, STATE_PAUSED)
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import PlatformNotReady
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+
+DOMAIN = 'dlna_dmr'
```

Comment:
The domain of this platform is media_player. Remove this constant. Instead define a constant to use when storing data in hass.data. Call i something with the platform name and end with `_DATA`.